client/src/components/grants/Card.tsx

```diff
@@ -22,11 +24,16 @@ function Card({ projectId }: { projectId: string }) {
     const logoImg = getProjectImage(loading, ImgTypes.logoImg, project);
 
     const { id } = getProjectURIComponents(projectId);
+    const projectChainId = projectId.split(":")[0];
```

Comment:
we can change `const { id } = getProjectURIComponents(projectId);`
to `const { chainId, id } = getProjectURIComponents(projectId);` to get the actual `projectChainId`